src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1766,3 +1760,23 @@ def list_locations(cmd, sku, linux_workers_enabled=None):
     client = web_client_factory(cmd.cli_ctx)
     full_sku = get_sku_name(sku)
     return client.list_geo_regions(full_sku, linux_workers_enabled)
+
+
+def _check_zip_deployment_status(deployment_status_url, authorization):
+    import requests
+    num_trials = 1
+    while num_trials < 200:
+        response = requests.get(deployment_status_url, headers=authorization)
+        res_dict = response.json()
```

Comment:
Do you need to wait a bit before poll it again? This would avoid stressing out the service end point, also 200 requests could be sent out within a second, is that long enough?

Reply:
i will add the wait- that a great suggestion i completely did not think of. 200 is the value i am using right now based on testing some Node apps with several packages, i can make this higher if needed. Ideally our zip deployments should be 1 min long, in case of react apps with a lot of packages i have seen this take ~5 mins which should be ideally fixed on our end for better performance